Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -360,6 +360,27 @@ static NSString *const STPSDKVersion = @"15.0.0";
 
 @end
 
+
+#pragma mark Payment Methods
+
+/**
+ STPAPIClient extensions for working with PaymentMethod objects.
+ */
+@interface STPAPIClient (PaymentMethods)
+
+/**
+ Creates a PaymentMethod object with the provided params object.
+ 
+ @see https://site-admin.stripe.com/docs/api/payment_methods/create
+ 
+ @param paymentMethodParams  The `STPPaymentMethodParams` to pass to `/payment_methods`.  Cannot be nil.
```

Comment:
do we always document endpoints like this instead of saying `/v1/payment_methods`?

Reply:
From what I can see it's not a convention, I just copied `confirmPaymentIntentWithParams`'s docs.  

I'll change to `/v1/payment_methods`